### PR TITLE
Add 'Copy element name' to context menu

### PR DESCRIPTION
**Pull request by LanceJernigan**
This feature was mentioned in issue #754 and should add a 'Copy element name' to the context menu.

### CODE REVIEW

File: agent/Bridge.js [hunk 1]
```diff
@@ -18,7 +18,7 @@ var performanceNow = require('fbjs/lib/performanceNow');
 
 // Use the polyfill if the function is not native implementation
 function getWindowFunction(name, polyfill): Function {
-  if (String(window[name]).indexOf(`[native code]`) === -1) {
+  if (String(window[name]).indexOf('[native code]') === -1) {
```
Comment (bvaughn) [inline agent/Bridge.js]: This was fixed in master yesterday. Maybe need to rebase?

File: frontend/Container.js [hunk 2]
```diff
@@ -133,6 +133,11 @@ var DEFAULT_MENU_ITEMS = {
         title: 'Show all ' + node.get('name'),
         action: () => store.changeSearch(node.get('name')),
       });
+      items.push({
+        key: 'copyNodeName',
+        title: 'Copy element name',
+        action: () => store.copyNodeName(node.get('name')),
+      });
```
Comment (bvaughn) [inline frontend/Container.js]: nit: I think this option should be at the bottom (after "Scroll to node")

I was also wondering if we should only show it for composite component types (eg `node.get('nodeType') === 'Composite'`)? It seems a bit unlikely that someone would want to copy "div" or "span" although I guess it doesn't hurt to have the menu option there either, if it makes things simpler.

File: frontend/ContextMenu.js [hunk 3]
```diff
@@ -11,12 +11,12 @@
...
 
 var React = require('react');
-var {sansSerif} = require('./Themes/Fonts');
+var { sansSerif } = require('./Themes/Fonts');
 var HighlightHover = require('./HighlightHover');
 
 var decorate = require('./decorate');
 
-import type {Theme} from './types';
+import type {Theme } from './types';
```
Comment (bvaughn) [inline frontend/ContextMenu.js]: This spacing is weird ^

I think we generally don't have spaces between the `{}`s and their inner words FWIW>

File: frontend/ContextMenu.js [hunk 4]
```diff
@@ -58,16 +63,39 @@ class ContextMenu extends React.Component {
     this.props.hideContextMenu();
   }
 
+  getHeight = element => {
```
Comment (bvaughn) [inline frontend/ContextMenu.js]: The changes to this file seem unrelated to the description of the PR. And I think they're actually included in another PR. Mind if we keep this PR focused on just the copy-name issue?

Comment (bvaughn) [inline frontend/ContextMenu.js]: I guess it's okay. We can just combine that PR and this PR if it's easier for you.

File: frontend/ContextMenu.js [hunk 5]
```diff
@@ -58,16 +63,39 @@ class ContextMenu extends React.Component {
...
+      pos.y -= elementHeight;
+    }
 
     if (!open) {
       return <div style={styles.hidden} />;
     }
 
     return (
-      <div style={styles.backdrop} onClick={this.handleBackdropClick}>
+      <div style={styles.backdrop} onClick={this.handleBackdropClick} ref={this.getHeight}>
```
Comment (bvaughn) [inline frontend/ContextMenu.js]: nit: Not a fan of the name of this method. It isn't a getter; it updates state. It's also a bit weird to have a ref-setter named "get" something. My preference would be to call it something like `_setRef`

File: frontend/ContextMenu.js (hunk 4 above)
Comment (LanceJernigan) [inline frontend/ContextMenu.js]: I'm honestly not sure how those even got added to this branch.  I didn't realize they were there until I made the pull request.

File: frontend/ContextMenu.js (hunk 3 above)
Comment (LanceJernigan) [inline frontend/ContextMenu.js]: The spacing should be fixed, I believe that was due to vscode's selection formatting.

File: frontend/ContextMenu.js (hunk 4 above)
Comment (bvaughn) [inline frontend/ContextMenu.js]: No worries. I assume you just branched from your other branch instead of master, maybe accidentally it sounds like 😁

**Comment by bvaughn** [comment]
I noticed a jumping issue that happens if you right click a node before first focusing on it (via left click):
context-menu-jump-alignment

This also happens in master though, and your alignment fix definitely improves the bottom overflow issue in master. So I think we can just tackle that in a follow up. (I mention it here just because I noticed it. Not because I expect you to fix it or anything. 😄 )

**Comment by LanceJernigan** [comment]
I noticed that, too.  I just assumed it was to scroll the element into view a little better instead of having it so close to the bottom of the container.  I wasn't sure if it was a bug or a feature.

**Comment by bvaughn** [comment]
Yeah, the scroll-into-view bit is a feature. It's just unfortunate how the jump can cause the node and context menu to be misaligned slightly. 😁  But not a big deal. This PR is a def improvement.

**Comment by LanceJernigan** [comment]
Gotcha.  If I have time, I'll try to look into a way to keep the two lined up a little better and submit a separate PR for it.

**Comment by bvaughn** [comment]
That would be awesome but only if you're interested and have the time. 🙇 
I'm almost done testing this. I think it looks good.

**Comment by bvaughn** [comment]
Thanks for this contribution!
